File: 10/RestaurantManagement/src/models/recipe.ts
export interface Recipe {
  id: number;
  author: string;
  name: string;
  type: string;
  recipe: string;
}

export const getEmptyRecipe = (): Recipe => ({
  id: -1,
  author: '',
  name: '',
  type: '',
  recipe: '',
});

File: 10/RestaurantAPI/RestaurantAPI/Models/Recipe.cs
namespace RestaurantAPI.Models
{
    public class RecipeDTO
    {
        public virtual long Id { get; set; }
        public virtual string? Author { get; set; }
        public virtual string? Name { get; set; }

        public virtual string? Type { get; set; }
        public virtual string? Recipe { get; set; }
    }
}
